presto-main/src/main/java/io/prestosql/SystemSessionProperties.java

```diff
@@ -111,6 +111,7 @@
     public static final String MAX_GROUPING_SETS = "max_grouping_sets";
```

Comment:
> `Collect plan statistics for explain only by default`

capitalize EXPLAIN

i read this as "for explain, only by default", while this change doesn't impact EXPLAIN at all
maybe

```
Skip stats collection for non-EXPLAIN queries by default
```